src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -165,10 +165,10 @@ const debounceFrame = (func: DebouncableFn): DebouncedFn => {
 };
 
 const createIconElements = (): { [key: string]: ReactElement<any> } => {
-	const iconList = ['vm', 'cloud', 'tag', 'inbox', 'check', 'loading', 'warning'];
+	const iconList = ['head', 'remote', 'tag', 'stash', 'check', 'loading', 'warning','added','modified','deleted','renamed','resolved'];
```

Comment:
Nitpick: consistent spacing after the commas within the array.

Reply:
Please install the recommended extensions for the GitLens repo (Prettier, ESLint, etc). With them everything should be auto formatted and linted on save.